src/router/validation/validationUtilities.test.ts

```diff
@@ -0,0 +1,41 @@
+import { InvalidResourceError, Validator } from 'fhir-works-on-aws-interface';
+import { validateResource } from './validationUtilities';
+
+describe('validateResource', () => {
+    const mockedSuccessValidator: Validator = {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        validate(resource: any) {
+            return Promise.resolve();
+        },
+    };
+    const mockedFailedValidatorA: Validator = {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        async validate(resource: any) {
+            // Sleep for one second, so if validatorA and validatorB are in validators array we can ensure validatorA
+            // takes longer to run than validator B. This is helpful for testing that the validators are run sequentially
+            await new Promise(r => setTimeout(r, 1000));
```

Comment:
I like the idea here, but you only need a couple milliseconds to make your point. Idling the tests for 1 second on every run is a bit much.

Reply:
updated to 10 ms